### ISSUE TITLE
Disable cudnn batch normalization

### ISSUE THREAD

**Issue by bit-scientist** [MCG-NJU/MOC-Detector#45]
You provided commands for disabling cudnn BN:
```
# PYTORCH=/path/to/pytorch # usually ~/anaconda3/envs/MOC/lib/python3.5.2/site-packages/
# for pytorch v0.4.1
sed -i "1254s/torch\.backends\.cudnn\.enabled/False/g" ${PYTORCH}/torch/nn/functional.py
```
I would like to be 100% sure that this only affects the cudnn environment-wise. Because you pointed out as a comment above., if I run this, I won't modify other `envs`' cudnn settings on my anaconda, right?

I had nightmares when I did something similar and wasted a couple days to reset everything back. I hope you got my point. Thanks.

**Comment by yixuanli98** [comment]
If you only modify the pytorch code in one of your anaconda environment, this will not affect your other environment.
